roles/osp_observability/templates/prometheus.yml.j2

```diff
@@ -1,18 +1,34 @@
-#jinja2: trim_blocks:False
 global:
   scrape_interval: {{ prometheus_scrape_interval }}
   external_labels:
-    {% if ansible_fqdn %}agent_host: {{ ansible_fqdn }}{% endif %}
-    {% if openstack_cloud_name %}cloud_name: {{ openstack_cloud_name }}{% endif %}
+    {% if ansible_fqdn -%}
```

Comment:
Will ansible_fqdn always be pre-defined? If not then you probably need to capture the situation where it wouldn't exist with a `if ansible_fqdn is defined and ansible_fqdn` as this will be fine if the variable is defined empty, but error if it just isn't defined at all.

Reply:
AFAIK this is always defined and never empty, so I'm not sure the purpose of the `if`. I could be mistaken, though, I was just making it valid yaml. If @paramite has no objection, I'm happy to remove the conditional.

Reply:
Seems safe enough for now. I'd probably just leave it.